codespell_lib/data/dictionary_code.txt

```diff
@@ -2,6 +2,8 @@ agrv->argv
 alloced->allocated
 amin->main
 apoint->appoint
+arange->arrange, a range,
+aranges->arranges
```

Comment:
I guess `arange` is for `numpy.arange`, but `aranges`?

Reply:
On github, most uses of `aranges` were related to DWARF. Should `aranges` go in dictionary_code or dictionary?

Reply:
Good question. I don't really know. I guess references to such DWARF details are rare enough to move `aranges` to the main dictionary. We can move it back to the code dictionary later on, if there are too many complaints.